src/reducers/books.js

```diff
@@ -0,0 +1,23 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = {
+  books: [
+    { bookId: Math.ceil(Math.random()), title: 'Understanding in C++', category: 'Programming' },
+    { bookId: Math.ceil(Math.random()), title: 'Baby names 2020', category: 'Kids' },
+    { bookId: Math.ceil(Math.random()), title: 'The Hunger', category: 'Horror' },
+    { bookId: Math.ceil(Math.random()), title: 'Beloved', category: 'Horror' },
+  ],
+};
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case CREATE_BOOK:
+      return { ...state, newBook: action.payload };
+    case REMOVE_BOOK:
+      return { ...state, removedBook: action.payload };
```

Comment:
* [ ] Remove book action should filter overbook and return all except the specified book that we intend to delete. Currently, it appears as if remove and add book are doing the same thing. Let us address this. we can return something like this `case 'REMOVE_BOOK': return state.filter(book => book.id !== action.id)`